### ISSUE TITLE
[A11yInspect] Fail: 1.3.1 A in /home/runner/work/test-app/test-app/src/main.tsx

### ISSUE THREAD

**Issue by sitaram-r** [sitaram-r/test-app#16]
## ❌ Accessibility Fail

**Rule:** 1.3.1 A
**File:** `/home/runner/work/test-app/test-app/src/main.tsx`
**Line:** 1, **Column:** 1

### Description
Missing heading level 1 (Minor)

---
*Generated by [A11yInspect](https://github.com/barrierbreak/vscode-scally)*
